disassemblers/ofrak_ghidra/ofrak_ghidra/ghidra_scripts/GetInstructions.java

```diff
@@ -118,7 +118,9 @@ class ResultInstruction {
             Register vle_register = instruction.getRegister("vle");
             if (vle_register != null) {
                 BigInteger vle_val = instruction.getValue(vle_register, false);
-                this.instruction_mode = vle_val.equals(BigInteger.ONE) ? "VLE" : this.instruction_mode ;
+                if (vle_val != null){
+                    this.instruction_mode = vle_val.equals(BigInteger.ONE) ? "VLE" : this.instruction_mode ;
+                }
```

Comment:
I would change it to either entirely use the ternary, or not use it at all.

```suggestion
                if (vle_val != null && vle_val.equals(BigInteger.ONE)){
                    this.instruction_mode = "VLE";
                }
```

or 

```suggestion
                this.instruction_mode = vle_val != null && vle_val.equals(BigInteger.ONE) ? "VLE" : this.instruction_mode ;
```